metric_scratch.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
```

Comment:
Is this `metric_scratch.py` function used for anything? We should remove it if not.